docs/05_integrations/django.md

```diff
@@ -114,3 +114,114 @@ class MyGraphQLView(GraphQLView):
 
 In this case we are doing the default processing of the result, but it can
 be tweaked based on your needs.
+
+# Async Django
+
+Strawberry also provides an async view that you can use with Django 3.1+
+
+```python
+from django.urls import path
+
+from strawberry.django.views import AsyncGraphQLView
+
+from api.schema import schema
+
+urlpatterns = [
+    path("graphql/", AsyncGraphQLView.as_view(schema=schema)),
+]
+```
+
+You'd also need to add `strawberry.django` to the `INSTALLED_APPS` of your
+project, this is needed to provide the template for the GraphiQL interface.
+
+## Options
+
+The `AsyncGraphQLView` accepts two options at the moment:
+
+- schema: mandatory, the schema created by `strawberry.Schema`.
+- graphiql: optional, defaults to `True`, whether to enable the GraphiQL
+  interface.
+
+## Extending the view
+
+We allow to extend the base `AsyncGraphQLView`, by overriding the following methods:
+
+- `async get_context(self, request: HttpRequest) -> Any`
+- `async get_root_value(self, request: HttpRequest) -> Any`
+- `async process_result(self, request: HttpRequest, result: ExecutionResult) -> GraphQLHTTPResponse`
+
+## get_context
+
+`get_context` allows to provide a custom context object that can be used in your
+resolver. You can return anything here, by default we return a dictionary with
+the request.
+
+```python
+class MyGraphQLView(AsyncGraphQLView):
+    async def get_context(self, request: HttpRequest) -> Any:
+        return {"example": 1}
+
+
+@strawberry.type
+class Query:
+    @strawberry.field
+    def example(self, info) -> str:
+        return str(info.context["example"])
+```
+
+Here we are returning a custom context dictionary that contains only one item
+called "example".
+
+Then we use the context in a resolver, the resolver will return "1" in this
+case.
```

Comment:
Using the `grapqhl+response` code block might be nice here to show the result rather than just describing it.